gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-healthcheck/src/main/java/io/gravitee/gateway/services/healthcheck/EndpointHealthcheckResolver.java

```diff
@@ -160,7 +160,12 @@ public void afterPropertiesSet() throws Exception {
     private HttpEndpoint convertToHttpEndpoint(Endpoint endpoint) {
         if (isHttpEndpoint(endpoint)) {
             try {
-                return mapper.readValue(endpoint.getConfiguration(), HttpEndpoint.class);
+                // FIXME: This creates a new instance of HttpEndpoint (and so definition.model.Endpoint) instead of using
+                // the one from the ManagedEndpoint. This is a temporary fix that need to be addressed, for details see
+                // https://github.com/gravitee-io/issues/issues/6437
```

Comment:
we can add this PR link to the mentioned issue also ?

Reply:
Done